lib/src/test/java/vmath/Mat3Test.java

```diff
@@ -9,4 +9,29 @@ public class Mat3Test {
     void givenNoArgs_whenMat3Created_thenMatrixIsIdentity() {
         //TODO
     }
+
+    @Test
+    void givenIndices_whenGetCalled_thenReturnCorrectValue() {
+        var m1 = new Mat3();
+
+        assertEquals(1.0f, m1.get(0, 0));
+        assertEquals(0.0f, m1.get(0, 1));
+        assertEquals(0.0f, m1.get(0, 2));
+        assertEquals(0.0f, m1.get(1, 0));
+        assertEquals(1.0f, m1.get(1, 1));
+        assertEquals(0.0f, m1.get(1, 2));
+        assertEquals(0.0f, m1.get(2, 0));
+        assertEquals(0.0f, m1.get(2, 1));
+        assertEquals(1.0f, m1.get(2, 2));
+    }
```

Comment:
The contents of this test would be better suited to the 'givenNoArgs_whenMat3Created_thenMatrixIsIdentity()' test, as that test is required to complete previous work and can also serve as a preliminary test of get().